docs/utilities/arrays.md

```diff
@@ -1,67 +1,65 @@
-# Array Utilities
-This document describes how to use array utilities in `pkg/shared/utils` folder
+# Slice Utilities
```

Comment:
Can we rename the file to `slices.md` too?

Reply:
I will rename it in another PR since that will require changing all the references.

Reply:
:+1: